app/config/FrontendAppConfig.scala

```diff
@@ -54,7 +54,6 @@ class FrontendAppConfig @Inject() (configuration: Configuration, servicesConfig:
 
   def signOutUrl(lrn: LocalReferenceNumber): String = s"$departureHubUrl/$lrn/delete-lock"
 
-  lazy val maxPreviousDocuments: Int   = configuration.get[Int]("limits.maxPreviousDocuments")
   lazy val maxSupportingDocuments: Int = configuration.get[Int]("limits.maxSupportingDocuments")
```

Comment:
To keep things consistent we could have something like
```
def maxPreviousDocuments(implicit phaseConfig: PhaseConfig) = phaseConfig.maxPreviousDocuments(configuration)
```
Then we could override this value in the phase config with
```
override def maxPreviousDocuments(configuration: Configuration): Int = configuration.get[Int]("limits.maxPreviousDocuments.transition")
```
and the same for post transition